src/redux/actions/index.js

```diff
@@ -0,0 +1,23 @@
+export const FETCH_POSTS_START = 'FETCH_POSTS_START';
+export const FETCH_POSTS_SUCCESS = 'FETCH_POSTS_SUCCESS';
+export const FETCH_POSTS_FAILURE = 'FETCH_POSTS_FAILURE';
+
+export const FETCH_POST_DETAILS_START = 'FETCH_POST_DETAILS_START';
+export const FETCH_POST_DETAILS_SUCCESS = 'FETCH_POST_DETAILS_SUCCESS';
+export const FETCH_POST_DETAILS_FAILURE = 'FETCH_POST_DETAILS_FAILURE';
+
```

Comment:
can you tell why having three actions aginst each step is a good idea, for loading purpose you could have used https://www.npmjs.com/package/react-redux-loading